mmcv/parallel/distributed.py

```diff
@@ -44,8 +44,15 @@ def train_step(self, *inputs, **kwargs):
                 'Reducer buckets have been rebuilt in this iteration.',
                 logger='mmcv')
 
-        if getattr(self, 'require_forward_param_sync', True):
-            self._sync_params()
+        if ('parrots' not in TORCH_VERSION
+                and digit_version(TORCH_VERSION) >= digit_version('1.11.0')):
+            if self._check_sync_bufs_pre_fwd():
+                self._sync_buffers()
+        else:
+            if (getattr(self, 'require_forward_param_sync', False)
+                    and self.require_forward_param_sync):
```

Comment:
https://github.com/pytorch/pytorch/blob/50c90a22be3ee6a547ad0222951f2c9f50c02b50/torch/nn/parallel/distributed.py#L277

``require_forward_param_sync`` is a attribute of ``DDP`` so do we still need to use `getattr` to get it rather than ``self.require_forward_param_sync``?

Reply:
It works in parrots. LGTM.

Reply:
And parrots also support `self.require_forward_param_sync`.